priv/repo/migrations/20200723112205_create_extension_timescaledb.exs

```diff
@@ -7,27 +7,28 @@ defmodule Membrane.Telemetry.TimescaleDB.Repo.Migrations.CreateExtensionTimescal
   @chunk_compress_policy_interval Application.get_env(:membrane_timescaledb_reporter,Repo)[:chunk_compress_policy_interval] || "1 minute"
 
   def up() do
```

Comment:
Why editing migrations instead of creating new one?

Reply:
I decided that there are not many people using the package to be honest (if any) and we have no benefit of keeping things compatible for now. Deleting the migration you did in one of the PRs makes it  look way more cleaner.

Reply:
That's true. It was hard to do it properly because of unchangeable `timescaledb.compress` :c okay, we can have it edited